runscript.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
@kylehuntsman what does this file do?

Reply:
Yeah, this is just for running any script without all of the testing setup that loadtest.sh sets up.

Reply:
I've added a README commit explaining how to use the runscript.sh command